src/k3s/mod.rs

```diff
@@ -0,0 +1,160 @@
+use std::collections::HashMap;
+use std::io;
+use std::io::ErrorKind;
+use std::path::{Path, PathBuf};
+
+use testcontainers::core::{Mount, WaitFor};
+use testcontainers::{Image, ImageArgs};
+
+const NAME: &str = "rancher/k3s";
+const TAG: &str = "v1.28.8-k3s1";
+pub const TRAEFIK_HTTP: u16 = 80;
+pub const KUBE_SECURE_PORT: u16 = 6443;
+pub const RANCHER_WEBHOOK_PORT: u16 = 8443;
+
+#[derive(Debug, Default, Clone)]
+pub struct K3s {
```

Comment:
Could you add a simple documentation with references to k3s and docker image? Example: https://github.com/testcontainers/testcontainers-rs-modules-community/blob/06efbb0474d1718efa459c718eff885e9354e833/src/rabbitmq/mod.rs#L6-L28

Reply:
I added a bit of documentation and an example like the rabbitmq module you linked